Doctrine/Phpcr/MediaManager.php

```diff
@@ -133,8 +134,12 @@ public function setDefaults(MediaInterface $media, $parentPath = null)
      */
     public function mapPathToId($path, $rootPath = null)
     {
-        // The path is being the id
-        $id = PathHelper::absolutizePath($path, '/');
+        try {
+            // The path is being the id
+            $id = PathHelper::absolutizePath($path, '/');
+        } catch (RepositoryException $e) {
+            throw new \OutOfBoundsException($e->getMessage());
```

Comment:
is this not a not found exception for us? maybe we could first validate the path and then throw the not found if the path is invalid, and then not add further catch handling to not hide other problems.